application/helpers/admin/import_helper.php

```diff
@@ -2008,6 +2008,7 @@ function XMLImportSurvey($sFullFilePath, $sXMLdata = null, $sNewSurveyName = nul
 
     // Import assessments --------------------------------------------------------
     if (isset($xml->assessments)) {
+        $aASIDReplacements = [];
```

Comment:
i dont think this is a good idea it will always set the array to empty also the ocnditions are clearly expecting a value here

Reply:
Sure, but the important thing is that the array initialization happens _outside_ the loop, which id does. So should be ok?